calculator-app/build.gradle.kts

```
@@ -45,15 +45,23 @@ dependencies {
     implementation(project(":calculator-domain"))
 
     implementation("org.jetbrains.kotlin:kotlin-stdlib:${Version.kotlin}")
-    implementation("androidx.core:core-ktx:1.9.0")
-    implementation("androidx.appcompat:appcompat:1.6.0")
-    implementation("com.google.android.material:material:1.7.0")
+    implementation("androidx.core:core-ktx:1.10.1")
+    implementation("androidx.appcompat:appcompat:1.6.1")
+    implementation("com.google.android.material:material:1.9.0")
     implementation("androidx.constraintlayout:constraintlayout:2.1.4")
-    implementation("androidx.fragment:fragment-ktx:1.5.5")
+    implementation("androidx.fragment:fragment-ktx:1.6.1")
+    implementation(project(mapOf("path" to ":calculator-data")))
```

다음과 같이 선언하는 것으로 충분해보입니다.
```suggestion
    implementation(project(":calculator-data"))
```